OpenRA.Mods.RA/Traits/Harvester.cs

```diff
@@ -137,11 +137,8 @@ Actor ClosestProc(Actor self, Actor ignore)
 						return occupancy * 12;
 					}));
 
-			// Reverse the found-path to find the refinery location instead of our location:
-			path.Reverse();
-
```

Comment:
Removing path.Reverse is intended. The code wants to look at the last element in the path and does so by reversing it and looking at the first element after that. Whilst that works, it's obviously more sensible to just directly look at the last element. The change a couple of lines down does that.

Unless you intended to mark another line?